README.md

```diff
@@ -44,8 +44,18 @@ and all subdirectories, and then check if there are cycles.
 If there are any, it will exit non-zero and print the files that are part of the cycle.
 Otherwise, it will not print anything and exit with zero.
 
-However, this is (currently) not configurable in any way,
-hence it is recommended to use the programmatic approach as described below.
+If you wish to overwrite the default settings you can pass the following parameters:
+
+```bash
+--checkDirectories #comma-separated list of absolute paths
```

Comment:
I think it might make more sense to have them relative to the `process.cwd()`, no?
That way, one could also provide an npm-script that already has the correct path relative to the package.json (with the assumption that "cwd = dir of package.json", which is a rather safe assumption to make)